### ISSUE TITLE
Simplify the ingest process and make source of monarch mp-hp-etc mappings more transparent

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/monarch-mapping-commons#61]
Currently we have this Makefile process:

https://github.com/monarch-initiative/monarch-mapping-commons/blob/e8dc36fcaf0375da1c653918268a5dc7747e6b36/monarch_mapping_commons.Makefile#L61-L69

This is not documented, but it seems to grab a CSV of unknown provenance from https://bbop-ontologies.s3.amazonaws.com/upheno/current/upheno-release/all/upheno_species_lexical.csv and run a bespoke mapping process.

This file doesn't seem to have been updated since 2022:

```
curl -I  https://bbop-ontologies.s3.amazonaws.com/upheno/current/upheno-release/all/upheno_species_lexical.csv

HTTP/1.1 200 OK
x-amz-id-2: JvpiirNXlumH4JRMltKcm8FrdDzmON8hMwyFsBofTV0QpujZHle2RH9gz31PuHSex/DjRtGq0C6aG4WPEc4qCQ==
x-amz-request-id: DXVS5HMZHFRAK1BA
Date: Fri, 06 Sep 2024 22:24:27 GMT
Last-Modified: Sun, 19 Jun 2022 18:33:20 GMT
ETag: "8433a4c273d87bd4f5d268989dd5d264-4"
Accept-Ranges: bytes
Content-Type: text/csv
Server: AmazonS3
Content-Length: 29130561
```

Same for
https://bbop-ontologies.s3.amazonaws.com/upheno/current/upheno-release/all/upheno_mapping_logical.csv

After this a bespoke mapping process using https://github.com/souzadevinicius/phenio-toolkit is used to make the sssom file used in monarch (see also https://github.com/obophenotype/upheno-dev/issues/53)

See also https://github.com/obophenotype/upheno/issues/962

**Comment by matentzn** [comment]
Before things become easier I documented the entire complexity here:

https://github.com/obophenotype/upheno/pull/963

Working on simplifying HP MP mappings there.

**Comment by matentzn** [comment]
I have addressed the complexity of the pipeline in #63, but this does not automatically increase HP-MP mapping transparency!

**Comment by matentzn** [comment]
pipeline is simplified, but will leave a bit open until my big refactoting is finished #63
